core/tests/webdriverio_utils/ContributorDashboardPage.js

```diff
@@ -251,6 +267,23 @@ var ContributorDashboardPage = function() {
       'Submit question tab button', submitQuestionTabButton);
     await this.waitForOpportunitiesToLoad();
   };
+
+  this.navigateToMyContributionTab = async function() {
+    await action.click(
+      'Submit question tab button', myContributionTabButton);
+    await this.waitForOpportunitiesToLoad();
+  };
+
+  this.selectTranslationReviewTab = async function() {
+    var reviewTabs = await reviewTabsSelector();
+    await action.click(
+      'Open translation review opportunity', reviewTabs[1]
```

Comment:
The description should be the name of the button/element. Maybe "Review tab button"

Reply:
Updated. PTAL.